smbc-digital/iag-webapp
# feat(accessibility_links): rm no underline

Jira [D-6375](https://stockportbi.atlassian.net/browse/DIGITAL-6375)
default is now underline on hover

After a conversation and review with Laura Hadley in attendance, she felt happy that links should be underlined by default. She saw the app with underline by default, but no hover-state colour changes.

We removed the style rule that was forcing it to be "text-decoration: none", to make sure the default hover state for links is now underlined.

When the underline is present on hover, the colour that was suggested in the card doesn't allow for a visible change.

Running the site improve browser extension with underline as default, passed. The previous link error was not present.

To test:

Pull the branch, launch ContentAPI & WebApp.

-) Hover over the links, and see that they have a hover state of underline, in the header, body, and footer.
-) Run the site improve tool, and see no issues to do with the link hover state, as previously mentioned ( in the Jira Card )
